backend/src/main/java/gov/cdc/usds/simplereport/config/simplereport/SmartyStreetsConfig.java

```diff
@@ -0,0 +1,25 @@
+package gov.cdc.usds.simplereport.config.simplereport;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.ConstructorBinding;
+
+@ConfigurationProperties(prefix = "simple-report.address-validation")
+public final class SmartyStreetsConfig {
```

Comment:
Further thoughts on property-holder classes:
* they should not say they are configuration, because they are properties. They probably should say they are properties (AdminEmailList is an interesting one, but we should solve that problem by making it go away)
* they should probably all reside in the same subpackage, which some but not all of them now do
* that package should probably be called `props` or `properties` rather than `simplereport`